plugin/skywalking/plugin-skywalking_test.go

```diff
@@ -0,0 +1,66 @@
+package skywalking
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/OpenFunction/functions-framework-go/framework"
+	"github.com/OpenFunction/functions-framework-go/plugin"
+)
+
+// HelloWorld writes "Hello, World!" to the HTTP response.
+func HelloWorldWithHttp(w http.ResponseWriter, r *http.Request) error {
+	fmt.Fprint(w, "Hello, World!\n")
+	return nil
+}
+
+func TestHTTP(t *testing.T) {
+	t.Setenv("FUNC_CONTEXT", `{
+  "name": "function-test",
+  "version": "v1.0.0",
+  "runtime": "Knative",
+  "port": "12345",
+  "prePlugins": [],
+  "postPlugins": [],
+  "pluginsTracing": {
+    "enable": true,
+    "provider": {
+      "name": "skywalking",
+      "oapServer": "127.0.0.1:11800"
+    },
+    "tags": {
+      "func": "function-test",
+      "layer": "faas",
+      "tag1": "value1",
+      "tag2": "value2"
+    },
+    "baggage": {
+      "key": "sw8-correlation",
+      "value": "base64(string key):base64(string value),base64(string key2):base64(string value2)"
+    }
+  }
+}
+`)
+	t.Setenv("POD_NAME", "prometheus-node-exporter-vhct4")
```

Comment:
If the function name is `function-test`, maybe the `POD_NAME` could be called `function-test-vhct4`?

Reply:
Done